src/DotVVM.Framework.Hosting.Owin/AppBuilderExtensions.cs

```diff
@@ -125,6 +124,7 @@ private static DotvvmConfiguration UseDotVVM(this IAppBuilder app, string applic
             startupTracer.TraceEvent(StartupTracingConstants.DotvvmConfigurationUserConfigureFinished);
 
             modifyConfiguration?.Invoke(config);
+            config.Development.Apply(config);
```

Comment:
Please change Development to Diagnostics

Reply:
Done. However rather unexpectedly, the `DotvvmDiagnosticsConfiguration` already exists (for 6 days already). It is in a different namespace but it might still cause confusion.

@exyi Can those two be merged somehow?

Reply:
I guess you mean this: https://github.com/riganti/dotvvm/blob/main/src/DotVVM.Framework/Diagnostics/DotvvmDiagnosticsConfiguration.cs

It exists for more than 6 days, from 2017 actually. Anyway, it is public, but seems to be only for internal use, so feel free to rename it (and maybe make it internal)